app/(chat)/api/chat/route.ts

```
@@ -83,53 +91,83 @@ export async function POST(request: Request) {
       ],
     });
 
+    // Determine which transport to use
+    let transport;
```

Biome (1.9.4)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)